test/powershell/engine/Help/HelpSystem.Tests.ps1

```diff
@@ -225,4 +225,38 @@ Describe "Get-Help should find help info within help files" -Tags @('CI', 'Requi
             Remove-Item $helpFilePath -Force -ErrorAction SilentlyContinue
         }
     }
+
+    # There is a bug specified to Tranvis CI that hangs the test if "get-help" is used to search pattern string. This doesn't repro locally. 
+    # This occurs even if Unix system just returns "Directory.GetFiles(path, pattern);" as the windows' code does.
+    # Since there's currently no way to get the vm from Tranvis CI and the test PASSES locally on both Ubuntu and MacOS, excluding pattern test under Unix system.
+    It "Get-Help should find pattern help files" -skip: (-not $IsWindows){
+        $helpFile1 = "about_testCase1.help.txt"
+        $helpFile2 = "about_testCase.2.help.txt"
+        $culture = (Get-Culture).Name
+        $helpFolderPath = Join-Path $PSHOME $culture
+        $helpFilePath1 = Join-Path $helpFolderPath $helpFile1
+        $helpFilePath2 = Join-Path $helpFolderPath $helpFile2
+
+        if (!(Test-Path $helpFolderPath))
```

Comment:
The if block can be replace by the following. If it already exists, it is not modified. 
```Powershell
$null = New-Item -ItemType Directory -Path $helpFolderPath -Force
```